server/clientRoutes.js

```diff
@@ -117,15 +128,78 @@ function clientsRouter(clients, defaultId) {
   });
 
   // get info about a specific client
-  router.get('/:id', (req, res) => {
+  router.get('/:id', async (req, res) => {
+    let configurations;
+    try {
+      configurations = await getConfig(req.params.id);
+    } catch (e) {
+      logger.error(e);
+      if (e.code === 'ENOENT')
+        return res.status(404).send(`Config for "${req.params.id}" not found`);
+      else
+        return res.status(500).send(`There was a problem with your request.`);
+    }
+
     const info = {
-      network: config.str('network'),
-      chain: config.str('chain', 'bitcoin')
+      configs: configurations.data
     };
+
+    if (req.query.checkStatus) {
```

Comment:
maybe instead of `checkStatus` it should be `?status=true`

Reply:
that's what I originally had actually, but when actually using it, it didn't feel correct (maybe not RESTful?) since it felt like the command was saying "GET this client by the id of "X" where status is true" rather than "return a status check". But that could be subjective so I don't know now. If you think `status` is better, I'm happy to go back.

Reply:
Maybe `?details=true` is more RESTful? GET this resource and return a detailed view about it

The two word camelcasing in the query param feels kind of clunky

Reply:
`details` isn't quite right either. You're already getting details, this is just specifying a specific detail that is not requested by default (since it has to do other async operations and so is slower).

Reply:
`verbose` is better, though I feel like I would prefer `status` to that just because it is explicit about what it's asking for.